infra/Makefile

```
@@ -1,86 +1,37 @@
 .PHONY: help build run test test-coverage fmt lint migrate-up migrate-down migrate-status docker-up docker-down clean test-local docker-prod-up generate-secrets create-env test-db fix-mysql-permissions setup-env-files
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)